lib/extension/listMPU.extension.js

```diff
@@ -1,9 +1,14 @@
 'use strict'; // eslint-disable-line strict
 
+const checkLimit = require('./tools').checkLimit;
+const DEFAULT_MAX_KEYS = 1000;
```

Comment:
The limit of 1000 comes from the AWS documentation, it's an hard limit.
But, the simple list extension is not intended to be used by S3 to do an object listing, bucketClient enforce the use of delimiter or listMPU extension, the limit of 10000 is here to avoid to use more RAM than the process can use.